paper/paper.Rmd

```diff
@@ -7,16 +7,16 @@ authors:
     affiliation: 1
     orcid: 0000-0002-2739-7082
   - name: Florent Angly
-    affiliation: ~
-    orcid: ~
+    affiliation: 6
+    orcid: 0000-0002-8999-0738
   - name: Michael Chirico
     affiliation: 2
     orcid: 0000-0003-0787-087X
   - name: Russ Hyde
     affiliation: 5
     orcid: ~
   - name: Ren Kun
-    affiliation: ~
+    affiliation: 7
```

Comment:
@renkun-ken Can you please let us know what's your current affiliation? I couldn't figure it out from [your website](https://renkun.me/about/).

Reply:
recommend emailing directly to ask as well

Reply:
Good idea!

Reply:
I had written to Ren Kun, but still no response to my e-mail even after 2 weeks. I am not sure how to proceed here. 

Should we retain “Unknown” as the affiliation?

Reply:
It seems reasonable to me.